rmw_connext_dynamic_cpp/src/functions.cpp

```diff
@@ -35,12 +39,24 @@
 // is in the include/rosidl_typesupport_connext_cpp/impl folder.
 #include <rosidl_generator_cpp/message_type_support.hpp>
 
+#include <rmw/impl/cpp/macros.hpp>
+
 #include "rosidl_typesupport_introspection_cpp/field_types.hpp"
 #include "rosidl_typesupport_introspection_cpp/identifier.hpp"
 #include "rosidl_typesupport_introspection_cpp/message_introspection.hpp"
 #include "rosidl_typesupport_introspection_cpp/service_introspection.hpp"
 #include "rosidl_typesupport_introspection_cpp/visibility_control.h"
 
+ROSIDL_TYPESUPPORT_INTROSPECTION_CPP_LOCAL
+inline std::string
+_create_type_name(
+  const rosidl_typesupport_introspection_cpp::MessageMembers * members,
+  const std::string & sep)
+{
+  return std::string(members->package_name_) + "::" + sep + "::dds_::" + members->message_name_ +
+         "_";
```

Comment:
In spite of what I've said in the other one, I updated both instances in cc82120, though I think both the change I made and the alternative:

```c++
  return std::string(members->package_name_) + "::" + sep + "::dds_::" + members->message_name_ +
    "_";
```

don't look as good.